src/justenoughlibtexpdf.c

```diff
@@ -331,6 +331,7 @@ int pdf_metadata(lua_State *L) {
   texpdf_add_dict(p->info,
                texpdf_new_name(key),
                texpdf_new_string(value, len));
+  return 0;
```

Comment:
looks fine.  normally the "right" solution would be to just make the function void, but most in the file are int.

Reply:
Well caught, actually! The integer value returned by these C wrappers is the number of Lua values pushed on the stack (= returned to the Lua calling context[^1]) so returning 0 is clearly the right/proper solution here.

[^1]: As per the definition of `lua_CFunction`: "To return values to Lua, a C function just pushes them onto the stack, in direct order (the first result is pushed first), and returns the number of results."

Reply:
figured it was something like that. read a rule once in a programming book. "Assume the programmer knows what [they're] doing unless given good reason to think otherwise." definitely applies here; no one who knows what they're doing would've repeated so many seemingly useless return's of 0 for no reason. :-)